Cargo.toml

```diff
@@ -27,3 +27,6 @@ unicode-xid = "0.1"
 # When disabled: emulate the same API as the nightly compiler's proc_macro crate
 # but in a way that works on all stable compilers >=1.15.0.
 nightly = []
```

Comment:
I am going to add `nightly = ["proc-macro"]` so that `default-features = false, features = ["nightly"]` can work.